doc/signify.txt

```diff
@@ -68,10 +68,10 @@ The next time Sy gets run, the buffer will be marked as `inactive`, so it won't
 look for changes anymore.
 
 Or you set |g:signify_disable_by_default|, which registers all new buffers as
-`inactive`, and enable Sy on demand using |signify-:SignifyToggle|.
+`inactive`, and enable Sy on-demand using |signify-:SignifyToggle|.
```

Comment:
I think the old version isn't wrong. It's `enable on demand`, but `on-demand enabling`.